packages/editor/src/client/utility.ts

```diff
@@ -156,39 +156,8 @@ export function mapTableSortTypeToGraphQLSortOrder(sortType: SortType): SortOrde
   }
 }
 
-export const DEFAULT_TABLE_PAGE_SIZES = [
-  {
-    value: 10,
-    label: 10
-  },
-  {
-    value: 20,
-    label: 20
-  },
-  {
-    value: 50,
-    label: 50
-  },
-  {
-    value: 100,
-    label: 100
-  }
-]
-
-export const DEFAULT_TABLE_IMAGE_PAGE_SIZES = [
-  {
-    value: 5,
-    label: 5
-  },
-  {
-    value: 10,
-    label: 10
-  },
-  {
-    value: 15,
-    label: 15
-  }
-]
+export const DEFAULT_TABLE_PAGE_SIZES = [10, 20, 50, 100]
```

Comment:
Is now just an array of numbers: https://rsuitejs.com/guide/v5-features/#removed-code-table-pagination-code-and-enhanced-code-pagination-code